zh-CN/mapping_dev.zh_CN

```diff
@@ -1,44 +1,44 @@
 :*"Cancel"=取消
-:*"Add Branch"=创建分支
+:*"Add Branch"=添加分支
 :*"Discard"=丢弃
-:*"Git-Flow"=Git工作流
+:*"Git-Flow"=Git 流
```

Comment:
I don't think it needs to be modified here. Because this is everyone's recognized "Git工作流"

I will not point out one by one in other places, please confirm and update.

Reply:
Thanks, I've fixed it.